client/blocks/author-selector/index.jsx

```diff
@@ -25,9 +25,10 @@ class AuthorSelector extends React.Component {
 	static propTypes = {
 		siteId: PropTypes.number.isRequired,
 		onSelect: PropTypes.func,
-		exclude: PropTypes.arrayOf( PropTypes.number ),
+		exclude: PropTypes.oneOfType( [ PropTypes.arrayOf( PropTypes.number ), PropTypes.func ] ),
 		allowSingleUser: PropTypes.bool,
 		popoverPosition: PropTypes.string,
+		transformAuthor: PropTypes.func,
```

Comment:
README here, would be good to update: https://github.com/Automattic/wp-calypso/blob/master/client/blocks/author-selector/README.md

Reply:
👍 53b94f5ca5459b9f2d2be813d7517ee75aea0710